app/assets/stylesheets/app.scss

```diff
@@ -19,6 +19,18 @@ html {
 
 body {
   margin: 0;
+  position: relative;
+  padding-bottom: 209px;
+  min-height: 100vh;
+}
+
+@media only screen and (max-width: 500px) {
```

Comment:
Instead of `500px`, use a breakpoint that's already used `(max-width: 768px)`
Also, we need to move all these values to CSS variables soon

Reply:
Yeah yeah we can use the existing breakpoint.